recipes/xraydb/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xraydb" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/scikit-beam/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0d2899348ffd8914950d4d76f52e5e6446460d25411ab317d2c552da4c8016e0
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  script: cd python && python setup.py install
```

Comment:
I think the failure of applying the patch in the CI's is due to this "trick", i.e. the pwd is already in the `python/` dir.